src/pages/studyView/tabs/ClinicalDataTab.tsx

```diff
@@ -62,9 +64,13 @@ export class ClinicalDataTab extends React.Component<IClinicalDataTabTable, {}>
                 render: (data: { [id: string]: string }) => {
                     return <a href={getSampleViewUrl(data.studyId, data.sampleId)} target='_blank'>{data.sampleId}</a>
                 }
-            }, {
-                ...this.getDefaultColumnConfig('studyId', 'Cancer Study')
             }];
+
+            if (_.find(this.props.store.visibleAttributes, chartMeta => chartMeta.uniqueKey === UniqueKey.CANCER_STUDIES) !== undefined) {
```

Comment:
@dippindots i believe this change is yours.  does it look right?

Reply:
@alisman I think this change belongs to @zhx828 .